City-Bureau/city-scrapers-wichita
# 🏗️ Build spider: WAMPO – Technical Advisory Committee

## What's this PR do?
Adds a spider to scrape meetings from the website of Wichita Area Metropolitan Planning Organization - Technical Advisory Committee. The new spider is called `wicks_wampo_tac`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl wicks_wampo_tac -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://www.wampo.org/technical-advisory-committee).

## Are there any smells or added technical debt to note?
This spider is virtually identical to this other WAMPO spider #15. However, I chose not to use a mixin because I think this may be the only other page on the WAMPO website that adheres to this particular structure and I don't think we have much guarantee the page layout will remain the same from year to year for either of them. To avoid optimizing too soon, I instead opted for simply a "cut and paste" approach for this spider.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206641237850843
